### PR TITLE
fix cert test to use manager namespace

### PR DESCRIPTION
**Pull request by RamLavi**
**What this PR does / why we need it**:
Currently the cert test uses specific manager namespace
that will only be correct if running on kubemacpool repo.
Let fix by using managerNamespace which gets the real namespace
from the cluster at  the beginning of the test suite.

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Review by phoracek — CHANGES_REQUESTED** [review]

File: tests/certificates_test.go
Comment (RamLavi) [reply tests/certificates_test.go:51]: Done

**Comment by qinqon** [comment]
/hold 
@RamLavi can you prepare a test PR at CNAO pointing to here to check tier1 lane ?

**Comment by RamLavi** [comment]
> /hold
> @RamLavi can you prepare a test PR at CNAO pointing to here to check tier1 lane ?

This fix is needed anyway, so I don't see a reason to add a PR to point to this PR..
so once it's merged I can retest one of my current open PRs on CNAO to make sure it  works.

Does that work for you?

**Pull request merged**
